app/frontend/private/pos/payment/index.html

```diff
@@ -41,9 +49,10 @@ <h2>Order Summary</h2>
                     <button class="num-btn" data-value="8">8</button>
                     <button class="num-btn" data-value="9">9</button>
                     <button class="num-btn" data-value="0">0</button>
-                    <button class="num-btn" data-value=".">.</button>
+                    <button id="delete"></button>
```

Comment:
Just one suggestion for this, I recommend replacing this with a "00" button, which basically multiplies the current amount by 100 for convenience